doc/articles/guides/xf-migration/toc.yml

```diff
@@ -0,0 +1,12 @@
+- name: Overview
+  href: overview.md
+- name: Uno Platform for Xamarin.Forms developers
+  href: intro.md
```

Comment:
You can (and should) use xref in the toc file eg
href: xref:Uno.XamarinFormsMigration.Intro